contracts/margin/external/ERC20/ERC20CappedLong.sol

```diff
@@ -33,7 +34,8 @@ import { ERC20Long } from "./ERC20Long.sol";
  */
 contract ERC20CappedLong is
     ERC20Long,
-    ERC20CappedPosition
+    ERC20CappedPosition,
+    DetailedERC20
```

Comment:
Ok so basically because you put this last in the inheritance chain `DetailedERC20` will override the functions (such as `name`, etc) that `ERC20Long` defines?

If so want to also confirm `DetailedERC20` does not override anything we don't want

Reply:
Yeah that's right.
https://github.com/OpenZeppelin/openzeppelin-solidity/blob/0e65947efbffc592cffea8c2ae9d3b8e11659854/contracts/token/ERC20/DetailedERC20.sol
Looks like it inherits only interfaces

Reply:
https://github.com/OpenZeppelin/openzeppelin-solidity/blob/0e65947efbffc592cffea8c2ae9d3b8e11659854/contracts/token/ERC20/DetailedERC20.sol

Looks like it does not